com.unity.netcode.gameobjects/Runtime/Messaging/MessagingSystem.cs

```diff
@@ -352,7 +352,7 @@ private bool CanSend(ulong clientId, Type messageType, NetworkDelivery delivery)
             return true;
         }
 
-        internal unsafe int SendMessage<TMessageType, TClientIdListType>(ref TMessageType message, NetworkDelivery delivery, in TClientIdListType clientIds)
+        internal int SendMessage<TMessageType, TClientIdListType>(ref TMessageType message, NetworkDelivery delivery, in TClientIdListType clientIds)
```

Comment:
Simply split the function in two parts. The first part doesn't need the `unsafe` keyword anymore